setup.py

```diff
@@ -94,10 +94,10 @@ def define_extensions(use_cython=False):
 
 def extract_gcc_binaries():
     """Try to find GCC on OSX for OpenMP support."""
-    patterns = ['/opt/local/bin/g++-mp-[0-9].[0-9]',
-                '/opt/local/bin/g++-mp-[0-9]',
-                '/usr/local/bin/g++-[0-9].[0-9]',
-                '/usr/local/bin/g++-[0-9]']
+    patterns = ['/opt/local/bin/g++-mp-([0-9]|10|11)$.([0-9]|10|11)$',
```

Comment:
maybe we can expand this pattern to allow future versions of gcc ->

something like (could filter down to valid version numbers)
glob.glob('/usr/local/bin/g++-[0-9]*')